src/components/pokemon/Pokemon.jsx

```diff
@@ -0,0 +1,58 @@
+import './Pokemon.css'
+import {useEffect, useState} from "react";
+import axios from "axios";
+
+function Pokemon({endpoint}) {
+    const [pokemon, setPokemon] = useState([])
```

Comment:
`pokemon` zal nooit een array zijn, maar een object. Ik zou zelf als default waarde `undefined` nemen. Dat is, net als `null`, een gangbare waarde voor iets wat nog ingeladen/opgehaald moet worden
```suggestion
    const [pokemon, setPokemon] = useState();
```